Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XldbDataStore.cs

```diff
@@ -86,7 +84,7 @@ public IEnumerable<string> GetEventsByType(int eventTypeID)
         /// <returns>Stored data in string format</returns>
         public string GetStoredData()

```

Comment:
since it's for testing only, make it `internal` and add `internalsVisibleTo` to dsc if needed